npm-debug.log

```diff
@@ -0,0 +1,45 @@
+0 info it worked if it ends with ok
```

Comment:
You seem to have added this extra `npm-debug.log` file in your commit. Make sure you remove this, as it would not be required. To see what I am talking about, you can click on that `Files Changed` button at the top of your PR. 
When you are adding files to a commit, you can run `git status` to see how many files have changed or been staged for the commit. If you changed multiple files, but only want to commit one, say `File1.md`. You can do that by running `add File.md` instead of `add .`. The `.` in the latter means that it adds _all_ the files that have changed.